README.md

```diff
@@ -3,13 +3,15 @@
 - **Version**:  1.0.0
 - **Certified**: No
 - **Publisher**: Fortinet
-- **Compatible Version**: FortiSOAR v7.3.0 and later
+- **Compatible Version**: FortiSOAR v7.4.0 and later
 
 # Overview
 
-The `SOC Experience` Solution Pack is to simplify the process of setting up a demo environment. Instead of manually configuring various components and settings, this solution pack initiates the setup process also provides set of playbooks that to automatically trigger scenarios for the deployed solution packs.
+The **SOC Experience** solution pack comes with pre-configured playbooks that automatically trigger scenarios in deployed solution packs. Instead of manually configuring various components and settings, this solution pack initiates the setup process and comes with a set of playbooks that automatically trigger scenarios for already deployed solution packs. 
```

Comment:
Overview should also cover the few SPs are also getting deployed as part of this SP and more info around it.